integrationExamples/gpt/hello_world.html

```diff
@@ -26,7 +26,9 @@
             bids: [{
                 bidder: 'appnexus',
                 params: {
-                    placementId: 13144370
+                    host: 'prebid',
+                    region: 'prebid-eu',
+                    traffic: 'banner'
```

Comment:
revert this change

Reply:
I have reverted the changes. I'm sorry for not having changed that.